### DISCUSSION TITLE
Icon Design

### DISCUSSION THREAD

**Discussion by XdiiasXD** [nguyenquyhy/Flight-Tracker-StreamDeck#52] [General]
Hi, 

I started using this plugin yesterday and I was very happy with the work that you guys did so i decided to make for you some buttons design that can be used in the A320 see if you like them, if so i can do some more :)
![AP OFF](https://user-images.githubusercontent.com/16303360/98948954-04213180-24ef-11eb-8f04-4e69628c0c57.png)
![AP ON](https://user-images.githubusercontent.com/16303360/98948955-04b9c800-24ef-11eb-93e9-82df16583124.png)
![APPR OFF](https://user-images.githubusercontent.com/16303360/98948957-05525e80-24ef-11eb-8c83-bcf889ca59f3.png)
![APPR ON](https://user-images.githubusercontent.com/16303360/98948961-05eaf500-24ef-11eb-9d22-eb2c870cae53.png)
![ATHR OFF](https://user-images.githubusercontent.com/16303360/98948962-05eaf500-24ef-11eb-8b11-1f23f8b4e46c.png)
![ATHR ON](https://user-images.githubusercontent.com/16303360/98948963-06838b80-24ef-11eb-9a49-3e049a74fa4b.png)

**Comment by nguyenquyhy** [comment]
Thanks a lot for sharing. I am working on a new feature where the images can be embedded in the profile, so I can share pre-build profiles here. The icon you shared would be very useful for that 👍.

**Comment by XdiiasXD** [comment]
@nguyenquyhy Do you want any help with design icons for the plugin?

**Comment by rug4ever** [comment]
Hello, Its possible i can get your complete list

**Comment by nguyenquyhy** [comment]
> @nguyenquyhy Do you want any help with design icons for the plugin?

I don't really plan to change the default icons, but I do plan to share community profiles (with custom icons) here.

**Comment by XdiiasXD** [comment]
@rug4ever that's all the icon that I have for now, but i can make some more for the community.

@nguyenquyhy alright if you need help with icons tell me because i'm very much pleased to help you guys.

**Comment by bagolu** [comment]
I made myself theses ones : 
![streamdeck_button_all2](https://user-images.githubusercontent.com/39214698/99429496-653e7000-2908-11eb-93a9-bc8ba597deb0.png)
![streamdeck_button_off](https://user-images.githubusercontent.com/39214698/99429498-65d70680-2908-11eb-831b-d60f15aaea87.png)
![streamdeck_button_off2](https://user-images.githubusercontent.com/39214698/99429499-65d70680-2908-11eb-961e-c6930afb7658.png)
![streamdeck_button_on](https://user-images.githubusercontent.com/39214698/99429501-666f9d00-2908-11eb-964e-ae35e5462308.png)
![streamdeck_button_on2](https://user-images.githubusercontent.com/39214698/99429502-666f9d00-2908-11eb-8a84-6f73baab1ace.png)
![streamdeck_button_up](https://user-images.githubusercontent.com/39214698/99429504-67083380-2908-11eb-9fca-2fabfd30c92b.png)

**Comment by Crunchmeister1** [comment]
> I made myself theses ones

Very nice. I could totally use those for my G1000 panel that I've pre-emptively built in anticipation of those variables being added in SimConnect.

**Comment by hey8you** [comment]
I have been making my own icons by taking screen shots. If anyone wants them I can post my gethub link.

![MS Flight Sim 2020 icons v2](https://user-images.githubusercontent.com/43597793/101535730-f32cda00-394d-11eb-959c-470ff6c21878.jpg)
![MS Flight Sim 2020 icons v3](https://user-images.githubusercontent.com/43597793/101536080-62a2c980-394e-11eb-99b9-5600e58d15be.jpg)

**Comment by phenDruck** [comment]
Oh yes I'd love to benefit from your hardwork!!! Thanks

**Comment by hey8you** [comment]
> Oh yes I'd love to benefit from your hardwork!!! Thanks

https://github.com/hey8you/Stream-Deck-and-MSFS-2020

**Comment by rmroc451** [comment]
moving to discussions
